lib/instrumentation/fastify.js

```diff
@@ -47,9 +47,12 @@ const setupMiddlewareHandlers = (shim, fastify) => {
     return function wrappedFastifyUser() {
       const args = shim.argsToArray.apply(shim, arguments)
       const middlewareFunction = args[0]
```

Comment:
You seem to be using destructuring everywhere else so just mentioning you could do that here too (but I don't care if you do or don't).